code/messaging/src/main/java/com/adobe/marketing/mobile/messaging/ContentCardImageManager.kt

```diff
@@ -72,7 +72,22 @@ object ContentCardImageManager {
         // Convert the InputStream to a Bitmap
         if (inputStream != null) {
             try {
-                completion(Result.success(BitmapFactory.decodeStream(inputStream)))
+                val bitmap = BitmapFactory.decodeStream(inputStream)
```

Comment:
- Can we flip the if clause to `if (inputStream == null) {` here and return early with completion failure for a clearer flow.
- We should also close the `inputstream` here after reading the content as a bitmap.